controls/C-0223-minimizeclusteraccesstoreadonlyforamazonecr.json

```diff
@@ -9,12 +9,13 @@
         "https://docs.aws.amazon.com/AmazonECR/latest/userguide/ECR_on_EKS.html"
     ],
     "attributes": {
-        "armoBuiltin": true
+        "armoBuiltin": true,
+        "useFromKubescapeVersion": "v2.2.5"
```

Comment:
this attribute is only needed on the rule level so no need to add it to the control (doesn't do any harm though)